test/test_evaluation.py

```diff
@@ -89,10 +89,28 @@ def test_optionvalues():
     assert(result == expected)
 
     result = session.evaluate("OptionValue[F, {a,b, l}]")
-    expected = session.evaluate('{89, 37, OptionValue[l]}')
+    expected = session.evaluate('{89, 37, l}')
     msg = "OptionValue::optnf: Option name l not found."
     assert result == expected, msg
 
+    session.evaluate('Options[f5]:={"a"->12}')
```

Comment:
It's a very minor thing, but the reducing (or DRYing) the repeated sequence of 
```
session.evaluate(...
result = ..
expected = ..
assert(result == expected)
```

is exactly what the loop in the other the other methods in this test are doing. 

I'd like to pull this code down though in a couple of days, and try it to understand it in more detail and try it out.

Reply:
Sure, I can contiene working on other stuff in the meantime